XLS-0033d-compact-fungible-tokens/README.md

```diff
@@ -175,16 +185,15 @@ The **`CFToken`** object represents an amount of a token held by an account that
 ##### 1.2.1.2.1 Fields
 A **`CFToken`** object can have the following fields. The key of each CFToken is stored in the Owner Directory for the account that holds the `CFToken`.
 
-| Field Name            | Required?          | JSON Type | Internal Type |
-| --------------------- |--------------------|-----------|---------------|
+| Field Name          | Required?          | JSON Type | Internal Type |
+|---------------------|--------------------|-----------|---------------|
 | `LedgerEntryType`   | :heavy_check_mark: | `number`  | `UINT16`      |
-| `Account`            | :heavy_check_mark: | `string`  | `ACCOUNTID`   |
-| `CFTokenIssuanceID`   | :heavy_check_mark: |  `string` | `UINT256`     |
-| `CFTAmount`              | :heavy_check_mark: |  `string` | `UINT64`      |
-| `LockedAmount`        | default            |  `string` | `UINT64`      |
-| `Flags`               | default            |  `number` | `UINT32`      |
-| `OwnerNode`               | default            |  `number` | `UINT64`      |
-| `CFTokenNode`               | default            |  `number` | `UINT64`      |
+| `Account`           | :heavy_check_mark: | `string`  | `ACCOUNTID`   |
+| `CFTokenIssuanceID` | :heavy_check_mark: | `string`  | `UINT256`     |
+| `CFTAmount`         | :heavy_check_mark: | `string`  | `UINT64`      |
+| `LockedAmount`      | default            | `string`  | `UINT64`      |
+| `Flags`             | default            | `number`  | `UINT32`      |
+| `OwnerNode`         | default            | `number`  | `UINT64`      |
```

Comment:
We should probably add PreviousTxnID and PreviousTxnLgrSeq to CFToken as well if we want to keep the ledger format uniform

Reply:
```suggestion
| `OwnerNode`         | default            | `number`  | `UINT64`      |
| `PreviousTxnID`     | :heavy_check_mark:  | `string`  | `HASH256`     |
| `PreviousTxnLgrSeq` | ️:heavy_check_mark: | `number`  | `UINT32`      |
```

I see new sections are added but maybe missed in the table 😃

Reply:
+1, I'll adjust.

Reply:
Fixed via c2a635cb377bee61ff8188eaa7f228e6ee22d49a